instructor/common.py

```diff
@@ -1670,6 +1670,40 @@ def facs_histograms_edit(request, assignment, sample_prep, sp):
     )
 
 
+@assignment_selected
+@check_assignment_owner
+@login_required
+def copy_histogram(request):
+    """
+    Copy a histogram from one mapping to a given list mappings
+    """
+    mapping_id = request.POST.get('copy_from_pk')
+    cell_treatment = request.POST.get('cell_treatment')
+    copy_to_list = json.loads(request.POST.get('copy_to_list'))
+    copy_from_mapping = get_object_or_404(
```

Comment:
We should also filter by assignment id here to make sure the user doesn't access data outside their assignment